makefile

```diff
@@ -1,7 +1,7 @@
 SRC := $(shell find src -name "*.d") \
 	$(shell find libdparse/src -name "*.d")
 INCLUDE_PATHS := -Ilibdparse/src -Isrc
-DMD_COMMON_FLAGS := -dip25 -w $(INCLUDE_PATHS)
+DMD_COMMON_FLAGS := -dip25 -w $(INCLUDE_PATHS) -Jviews
```

Comment:
Oh, I see now the "views" is for string inclusion. Why is it called "views" though? It looks like a funny name. But please squash the change adding the directory to the `.gitignore` file with this one.

Reply:
`views` is the default folder dub uses for string inclusion

Reply:
OK then. I wonder where does that comes from, but if it's some sort of defacto standard, I'm fine with it.

Reply:
It comes from the fact that vibed's `diet-templates` are in there.
which are the view in M-V-C.

Reply:
Typical "historical reasons" :man_facepalming: